charts/astronomer/templates/registry/registry-configmap.yaml

```diff
@@ -52,11 +52,15 @@ data:
       headers:
         X-Content-Type-Options: [nosniff]
     auth:
+      {{- if .Values.registry.enableInsecureReadingAndWriting }}
+      disabled: true # This disables authentication when the flag is set
```

Comment:
IMHO the comment doesn't add anything that isn't obvious in the code.
```suggestion
      disabled: true
```

Reply:
this change has implications we cannot do this

Reply:
We can't delete a comment?